app/controllers/api/status_controller.rb

```diff
@@ -26,3 +47,7 @@ def check
     })
   end
 end
+
```

Comment:
This should become `private` so that it can't be routed to accidentally.

Reply:
Deleting it.